ui/css/perf.css

```diff
@@ -493,6 +493,10 @@ li.pagination-active.active > button {
 }
 
 /* table in graphs view */
+.text-muted {
```

Comment:
Please remove this rule as this has no effect.

Reply:
ohh its left!